CPT331.Data.Parsers/QldXmlParser.cs

```diff
@@ -62,14 +64,12 @@ protected override void OnParse(string fileName, List<Crime> crimes)
 						count = Convert.ToInt32(countDouble);
 					}
 
-					//	Console.WriteLine($"{localGovernmentAreaName}: {offenceName} - {count}");
-
 					LocalGovernmentArea localGovernmentArea = localGovernmentAreas.Where(m => (m.Name.EqualsIgnoreCase(localGovernmentAreaName) == true)).FirstOrDefault();
 					Offence offence = null;
 
-					if (String.IsNullOrEmpty(offenceName) == false)
+					if ((String.IsNullOrEmpty(offenceName) == false) && (offences.ContainsKey(offenceName) == true))
```

Comment:
The Dictionary collection type and the Comparer methods seems to be repeated in the different parsers..,

Should we create a Offense Comparer class and reuse the Dictionary collection and the Add/ContainsKey methods?

This may prevent the need for .ToUpper() on every row add and comparison.